main.tf

```diff
@@ -142,7 +145,9 @@ module "operator" {
   operator_notification_protocol = var.operator_notification_protocol
   operator_notification_topic    = var.operator_notification_topic
 
+  # tags
   freeform_tags = var.freeform_tags["operator"]
+  defined_tags  = var.defined_tags["operator"]
```

Comment:
The operator module (https://github.com/oracle-terraform-modules/terraform-oci-operator) doesn't have `defined_tags` yet. Please either
1. Remove it for now and add it for operator when we've added `defined_tags` in the operator module.
2. Or implement `defined_tags` in the operator module. We'll cut a new release for it so we can then use this.

Reply:
As for the bastion tags, I removed them for the moment